dts/bindings/net/wireless/generic-coex-gpio.yaml

```diff
@@ -0,0 +1,18 @@
+description: |
+    Generic representation of Coexistance pinx for radio
+
+compatible: "generic-coex-gpio"
+
+include: base.yaml
+
+properties:
+  grant-gpios:
+    type: phandle-array
+    description: |
+        GPIO input for the external transceiver
+
+  grant-delay-us:
+    type: int
+    default: 150
+    description: |
+        Delay after assertion for the external transceiver operation
```

Comment:
default needs to be justified, per guidelines. If there's no way to justify it, make it required.

Reply:
Added required to true.